src/libs/actions/Report.js

```diff
@@ -1522,7 +1522,8 @@ function createPolicyRoom(policyID, reportName, visibility) {
             }
             return fetchChatReportsByIDs([response.reportID]);
         })
-        .then(([{reportID}]) => {
+        .then((response) => {
+            const {reportID} = _.values(response)[0];
```

Comment:
- Avoid object destructuring for a single variable that you only use once. [see style.md#destructuring](https://github.com/Expensify/App/blob/main/STYLE.md#destructuring)
- Use `_.first`for accessing the first element of the array.
- Use lodashGet for accessing `reportID`

Reply:
I think this will meet all the conditions. It will set reportID as false if there's no response resulting in early return.
`const reportID = lodashGet(_.first(_.values(chatReports)), 'reportID', false);`

Reply:
Sounds good

Reply:
@rushatgabhane, done ✔️